cime_config/stream_cdeps.py

```diff
@@ -336,14 +336,30 @@ def create_stream_xml(
                         ),
                     )
                     if var_key in valid_values:
+
```

Comment:
If you are expecting that the variable may or may not have surrounding quotes why not just strip the quotes if they exist and avoid asking the user for a correction?